.github/workflows/functional_tests.yml

```diff
@@ -6,14 +6,14 @@ on:
       - main
   workflow_dispatch:
 jobs:
-  single-transaction:
+  functional-tests:
     runs-on:
       group: github-v1
     env:
       CC: gcc
     steps:
       - name: Install ausearch for debugging seccomp/bad syscalls
-        run: sudo apt update && sudo apt install -y auditd
+        run: sudo apt update && sudo apt install -y auditd libudev-dev
```

Comment:
Think this dep can be removed now?

Reply:
But you're adding it as part of this change? What requires it? I think it was here when you built the `solana-keygen` command but isn't needed now?

Reply:
You're right good catch, updated.